library/src/main/java/com/yausername/youtubedl_android/YoutubeDLUpdater.java

```diff
@@ -22,7 +22,7 @@
     private YoutubeDLUpdater() {
     }
 
-    private static final String releasesUrl = "https://api.github.com/repos/yausername/youtubedl-lazy/releases/latest";
+    private static final String releasesUrl = "https://api.github.com/repos/xibr/ytdlp-lazy/releases/latest";
```

Comment:
I'm fine with this being changed. That way it will continue to work for users of the older version.